src/DemoUI/src/Views/resources/css/index.css

```diff
@@ -0,0 +1,1865 @@
+.pl-main-wrapper {
+    position: relative;
+    clear: both;
+    padding: 0 !important;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)